common/src/main/java/com/wynntils/features/combat/CustomLootrunBeaconsFeature.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.combat;
+
+import com.wynntils.core.consumers.features.Feature;
+import com.wynntils.core.persisted.Persisted;
+import com.wynntils.core.persisted.config.Category;
+import com.wynntils.core.persisted.config.Config;
+import com.wynntils.core.persisted.config.ConfigCategory;
+
+@ConfigCategory(Category.COMBAT)
+public class CustomLootrunBeaconsFeature extends Feature {
```

Comment:
What happens if you disable this feature? I found where you check the config about hiding beacons, but I assume that you will only draw the new beacons if this feature is enabled, right? I just can't figure out where that rendering is happening.

Reply:
`LootrunBeaconMarkerProvider#isEnabled`

Reply:
I see. I was looking for something in the map package; I forgot about how you designed the new providers. Thanks!